static/lang/en.json

```
@@ -3183,7 +3183,12 @@
         "SavesWillShort": "Will",
         "SavingThrowWithName": "{saveName} Saving Throw",
         "Scene": {
-            "Plural": "Scenes"
+            "HearingRange": {
+                "Caveat": "*For the performance purposes, the radius is a real measurement offset from a token's center rather than a grid-based one from its edges.",
```

```suggestion
                "Caveat": "*For performance purposes, the radius is a real measurement offset from a token's center rather than a grid-based one from its edges.",
```
real measurement offset?